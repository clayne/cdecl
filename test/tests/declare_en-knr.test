EXPECTED_EXIT=65
cdecl -xknr <<END
declare x as enum E
END
