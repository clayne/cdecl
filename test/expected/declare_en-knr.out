declare x as enum E
             ^
14: error: "enum": unsupported type until C89
